devWallets.md

```diff
@@ -125,6 +125,91 @@ async function handleStoreEvent() {
 
+## DID Authentication with CHAPI
+This section is written from the perspective of web wallets.  CHAPI provides a simple method for a 3rd party website to request an individual present their Decentralized Identifier (DID) and prove their identity.  The individual selects a digital wallet to respond to this DID Authentication request.
+
+### Resources
+- [Verifiable Presentation Request](https://w3c-ccg.github.io/vp-request-spec/#did-authentication)
+- [Verifiable Credentials Data Model](https://www.w3.org/TR/vc-data-model/#presentations)
+- [DID Core Specification](https://www.w3.org/TR/did-core/#authentication)
+
+#### 1. The 3rd Party Site sends a DID Authentication Request
+The individual interacts with a 3rd party website, triggering a request for _DID Authentication_.  The site sends a _Verifiable Presentation Request_ (VPR) using the CHAPI `get()` event.  
+
+An example VPR is shown below.  Like the other CHAPI examples on this site, the VPR is wrapped in a `web` credential object - this ensures that it is passed to a _Credential Handler_ in the individual's browser.
+```
+"web": {
+  "VerifiablePresentation": {
+      "query": {
+        "type": "DIDAuthentication"
+      },
+      "challenge": "IME0WNG2MIOsYsPgezxAM", //randomly-generated challenge string (e.g.,a UUID, nanoid, or bnid)
+      "domain": "https://playground.chapi.io" //URL of your web app (where the wallet will respond to the DID Auth request
+  },
+  "recommendedHandlerOrigins": [
+      "https://demo.vereswallet.dev/"
+  ]
+}
+```
+
+#### 2. The Digital Wallet responds with a Verifiable Presentation
+The individual selects a digital wallet, which responds to the CHAPI `get()` event.  The example code below shows two functions, plus a third function that your wallet will need to create
+
+- `handleGetEvent()` responds to the CHAPI `get()` event and calls `formDIDAuthResponse()` if it sees a request for DID Authentication.
+- `formDIDAuthResponse()` creates a "web" credential containing a signed Verifiable Presentation meeting the VPR Spec for DID Authentication
+- `signDidAuthPresentation()` is the lower-level function in your wallet that creates a signed Verifiable Presentation with the user's DID.
+
+```
+async function formDidAuthResponse({challenge, domain}) {
+    const dataType = 'VerifiablePresentation';
+
+    //Add your code for signing a verifiable presentation
+    didAuthPresentation = await signDidAuthPresentation({challenge, domain});
+
+    //wrap the DID Auth presentation in a web credential
+    const credentialType = 'VerifiablePresentation';
+    const didAuthResponse = new WebCredential(
+      credentialType, didAuthPresentation, {
+      recommendedHandlerOrigins: []
+    });
+
+    return didAuthResponse;
+}
+
+async function handleGetEvent() {
+    const event = await WebCredentialHandler.receiveCredentialEvent();
+    console.log('Store Credential Event:', event.type, event);
```

Comment:
Maybe this log message should say just "Credential Event" instead ... the example here is "get", not "store" anyway.